oncotator/datasources/GenericGeneDatasource.py

```diff
@@ -84,7 +84,7 @@ def annotate_mutation_given_index_value(self, value, mutation):
         #if any([c in mutation for c in self.output_headers]):
         for c in self.output_headers:
             if c in mutation:
-                raise Exception('Error: Non-unique header value in annotation table (%s)' % (c))
+                logging.warn('WARNING: Non-unique header value in annotation table (%s)' % (c))
```

Comment:
Should this being an error vs a warning depend on whether the mutation allows duplicate annotations?